src/sbb.py

```diff
@@ -52,8 +70,9 @@ def __init__(self, num_voters: int, twoM: int):
         self._num_voters: int = num_voters
         self._twoM: int = twoM
         self._ballot_receipts: List[str] = []
-        self._svr_commitments: List[str] = []
+        self._svr_commitments: List[List[Any]] = []
```

Comment:
You can make this more strict, `List[List[Dict[str, int]]]`

Reply:
👍